internal/v5/api_test.go

```diff
@@ -816,6 +834,25 @@ func (s *APISuite) TestMetaPermPutUnauthorized(c *gc.C) {
 	})
 }
 
+func (s *APISuite) TestMetaTerms(c *gc.C) {
```

Comment:
Please add a test for retrieving terms for a bundle too (to test whatever behaviour is decided upon)